tests/contract/test_parent_held_audit.py

```
@@ -21,6 +23,103 @@
+        if write_fd >= 0:
+            os.close(write_fd)
+        if proc.poll() is None:
+            os.killpg(proc.pid, signal.SIGKILL)
+            proc.wait(timeout=5)
+
+
+def test_parent_watchdog_accepts_a_valid_descriptor_above_1024():
+    read_fd, write_fd = os.pipe()
+    high_fd = fcntl.fcntl(read_fd, fcntl.F_DUPFD, 1025)
+    os.close(read_fd)
+    proc = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)